apps/app/src/views/MyReservations.vue

```diff
@@ -6,18 +6,29 @@
       :items="reservations"
       :items-length="reservations.length"
       :loading="isLoading">
+      <template #[`item.actions`]="{ item }">
+        <v-icon size="small" @click="deleteReservation(item)">
+          mdi-delete
+        </v-icon>
+        <ReservationDeleteModal
+          v-bind="{ reservation: selectedReservation }"
+          @cancel="closeDeleteModal()"
+          @delete="reservation => deleteReservationConfirm(reservation)" />
```

Comment:
Also, you can implement this so that the delete icon is a modal activator and then on delete confirmation, you perform delete action.
Ps. v-icon should be wrapped with a button tag because it is a clickable action.